backend/src/models/service.ts

```diff
@@ -20,15 +20,20 @@ const filterProducts = (product: Product) => {
     // https://github.com/AliasIO/wappalyzer/issues/3305
     return false;
   }
-  if (cpe === 'cpe:2.3:a:apache:coyote:1.1:') {
+  if (cpe === 'cpe:/a:apache:coyote:1.1:') {
     // Intrigue Ident incorrectly detects "Apache Coyote 1.1"
     // https://github.com/intrigueio/intrigue-ident/issues/51
     return false;
   }
-  if (cpe === 'cpe:2.3::generic:unauthorized::') {
+  if (cpe === 'cpe:/a::generic:unauthorized::') {
```

Comment:
Actually, intrigue ident creates cpe's prefixed by "cpe:2.3".

Reply:
We change the format before calling this though, right?